ui/src/pages/Alert/Shield/index.tsx

```diff
@@ -289,7 +293,7 @@ export default function Shield() {
           </h2>
         }
         extra={
-          access.alarmwrite ? (
+          access.alarmwrite || !access.obclusterread ? (
```

Comment:
Why can a user create a shield without `obcluster:read` permission? I thought `access.alarmwrite && access.obclusterread` should be correct.